ansible/monitoring.yml

```diff
@@ -76,6 +76,18 @@
   hosts: prometheus
   tags: prometheus
   tasks:
+    - name: Check for existing prometheus binaries
+      stat:
+        path: /usr/local/bin/{{ item }}
```

Comment:
Might binaries ever be installed somewhere that wasn't `/usr/local/bin`?

Reply:
The install path is a "private" role variable (prefixed _) in the `cloudalchemy.prometheus` role so I think we can assume not, when using that role at least.

Reply:
Sounds sensible to me